packages/graph-engine/src/nodes/string/index.ts

```diff
@@ -8,7 +7,6 @@ import uppercase from "./uppercase.js";
 export const nodes = [
   lowercase,
   stringify,
-  joinString,
```

Comment:
This would also need to be noted as a major break

Reply:
This also feels like we should rather keep this join rather than have it under array. The array implementation feels too generic and is likely more incorrect than this which is specifically typed to only be strings

Reply:
moved